internal/utils/common.go

```diff
@@ -233,24 +234,3 @@ func GetHostProcCmdline() string {
 	}
 	return proc
 }
-
-// EfiBootFromInstall will try to check the /sys/firmware/efi/LoaderDevicePartUUID-4a67b082-0a4c-41cf-b6c7-440b29bb8c4f
-// systemd vendor Id is 4a67b082-0a4c-41cf-b6c7-440b29bb8c4f and will never change
-// LoaderDevicePartUUID contains the partition UUID of the EFI System Partition the boot loader was run from. Set by the boot loader.
-// This will return true if we are running from a DISK device, which sets the efivar
-// This wil return false when running from a volatile media, like CD or netboot as it cannot infer where it was booted from
-// Useful to check if we are on install phase or not
-// This efi var is VOLATILE so once we reboot is GONE. No way of keeping it across reboots, its set by the bootloader.
-func EfiBootFromInstall() bool {
```

Comment:
Moved [to the kairos-sdk](https://github.com/kairos-io/kairos-sdk/pull/69/files#diff-ad95c17c31b539a3e385a23d86bbecf4d8a925ba6679aec2e66b5202ba4c4f81R197)